### DISCUSSION TITLE
Customization: button from API to Product

### DISCUSSION THREAD

**Discussion by jpcalhau** [Azure/api-management-developer-portal#2595] [Feature request]
Would be nice if we could customize the developer portal by adding a button at the api-details page, by clicking this button, we would jump to product detail page (where that API belongs to).

**Comment by ghost** [comment]
@jpcalhau, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@jpcalhau, thank you for requesting this feature.
